test/e2e/map-modify-actions.test.js

```diff
@@ -15,120 +15,61 @@ describe('vl-map-modify-action', async () => {
 
     const layer = await action.getLayer();
     await getDriver().wait(async () => (await layer.getNumberOfFeatures()) === 1);
-
-    const polygonFeature = await layer.getFeature(1);
-    const polygonCoordinates = polygonFeature.geometry.coordinates;
-    assert.lengthOf(polygonCoordinates, 1);
-
-    const points = polygonCoordinates[0];
-    assert.lengthOf(points, 4);
-
+    const points = (await layer.getFeature(1)).geometry.coordinates[0];
     const gent = points[0];
-    assert.lengthOf(gent, 2);
-
     const mechelen = points[1];
-    assert.lengthOf(mechelen, 2);
-
     const antwerpen = points[2];
-    assert.lengthOf(antwerpen, 2);
-
     const leuven = points[3];
-    assert.lengthOf(leuven, 2);
 
     await action.movePointByCoordinates(gent, willebroek);
 
     await getDriver().wait(async () => (await layer.getNumberOfFeatures()) === 1);
-    features = await layer.getFeatures();
-    assert.lengthOf(features, 1);
-
-    const modifiedPolygonFeature = await layer.getFeature(1);
-    const modifiedPolygonCoordinates = modifiedPolygonFeature.geometry.coordinates;
-    assert.lengthOf(modifiedPolygonCoordinates, 1);
-
-    const modifiedPoints = modifiedPolygonCoordinates[0];
-    assert.lengthOf(modifiedPoints, 4);
-
-    const modifiedPoint = modifiedPoints[0];
-    assert.lengthOf(modifiedPoint, 2);
-    assert.notSameOrderedMembers(modifiedPoint, gent);
-
-    const unmodifiedPoint2 = modifiedPoints[1];
-    assert.lengthOf(unmodifiedPoint2, 2);
-    assert.sameOrderedMembers(unmodifiedPoint2, mechelen);
+    assert.lengthOf(await layer.getFeatures(), 1);
 
-    const unmodifiedPoint3 = modifiedPoints[2];
-    assert.lengthOf(unmodifiedPoint3, 2);
-    assert.sameOrderedMembers(unmodifiedPoint3, antwerpen);
-
-    const unmodifiedPoint4 = modifiedPoints[3];
-    assert.lengthOf(unmodifiedPoint4, 2);
-    assert.sameOrderedMembers(unmodifiedPoint4, leuven);
+    const modifiedGeometry = (await layer.getFeature(1)).geometry;
+    assert.include(modifiedGeometry, {type: 'Polygon'});
+    assert.includeDeepMembers(modifiedGeometry.coordinates[0], [mechelen, antwerpen, leuven]);
+    assert.notIncludeDeepMembers(modifiedGeometry.coordinates[0], [gent]);
```

Comment:
ik zou het zelfs nog eenvoudiger maken en gewoon checken dat de hele array van coordinaten van "before" de modify niet deep equal is aan die van er na ...

Reply:
heb het nog eens herschreven nu met gebruik van de deepEquals voor de 3 e2e testen